models/index.js

```diff
@@ -0,0 +1,24 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Move your module exports below your user init. You export before you define class, When i tested importing it the values were null.